src/styles/_catalog.scss

```
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms ease;
```

don't use all, it is better to clearly indicate what is being animated
```suggestion
    transition: transform 300ms ease;
```

---

добре, поняв